newIDE/app/src/LocalApp.js

```diff
@@ -48,60 +49,62 @@ export const create = (authentication: Authentication) => {
       eventsFunctionsExtensionOpener={LocalEventsFunctionsExtensionOpener}
     >
       {({ i18n }) => (
-        <ProjectStorageProviders
-          appArguments={appArguments}
-          storageProviders={[
-            LocalFileStorageProvider,
-            UrlStorageProvider,
-            CloudStorageProvider,
-          ]}
-          defaultStorageProvider={LocalFileStorageProvider}
-        >
-          {({
-            getStorageProviderOperations,
-            storageProviders,
-            initialFileMetadataToOpen,
-            getStorageProvider,
-          }) => (
-            <MainFrame
-              i18n={i18n}
-              renderMainMenu={(props, callbacks) => (
-                <ElectronMainMenu props={props} callbacks={callbacks} />
-              )}
-              renderPreviewLauncher={(props, ref) => (
-                <LocalPreviewLauncher {...props} ref={ref} />
-              )}
-              renderExportDialog={props => (
-                <ExportDialog
-                  project={props.project}
-                  onSaveProject={props.onSaveProject}
-                  onChangeSubscription={props.onChangeSubscription}
-                  onClose={props.onClose}
-                  automatedExporters={localAutomatedExporters}
-                  manualExporters={localManualExporters}
-                  onlineWebExporter={localOnlineWebExporter}
-                />
-              )}
-              renderGDJSDevelopmentWatcher={
-                isDev ? () => <LocalGDJSDevelopmentWatcher /> : null
-              }
-              storageProviders={storageProviders}
-              resourceMover={LocalResourceMover}
-              resourceFetcher={LocalResourceFetcher}
-              getStorageProviderOperations={getStorageProviderOperations}
-              getStorageProvider={getStorageProvider}
-              resourceSources={localResourceSources}
-              resourceExternalEditors={localResourceExternalEditors}
-              extensionsLoader={makeExtensionsLoader({
-                gd,
-                objectsEditorService: ObjectsEditorService,
-                objectsRenderingService: ObjectsRenderingService,
-                filterExamples: !isDev,
-              })}
-              initialFileMetadataToOpen={initialFileMetadataToOpen}
-            />
-          )}
-        </ProjectStorageProviders>
+        <ErrorBoundary title="GDevelop encountered an issue">
```

Comment:
What do you think of putting this inside `Providers`?

Reply:
I'll try!